File: applications/vehicle-dashboard/src/main/java/com/vmware/vehicle/dashboard/repository/VehicleRepository.java
package com.vmware.vehicle.dashboard.repository;

import com.vmware.tanzu.data.IoT.vehicles.domains.Vehicle;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface VehicleRepository extends CrudRepository<Vehicle,String> {
}

File: applications/iot-connected-vehicle-dashboard/src/main/kotlin/com/vmware/tanzu/data/IoT/connected/vehicle/dashboard/repositories/VehicleRepository.kt
package com.vmware.tanzu.data.IoT.connected.vehicle.dashboard.repositories

import com.vmware.tanzu.data.IoT.vehicles.domains.Vehicle
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository


/**
 * @author Gregory Green
 */
@Repository
interface VehicleRepository : CrudRepository<Vehicle,String> {
}